src/rules/converters/jsdoc-format.ts

```diff
@@ -0,0 +1,22 @@
+import { RuleConverter, ConvertedRuleChanges } from "../converter";
+
+export const JSDocNoticeMsg =
+    "ESLint does not support enabling check for multiline comments in the first line";
```

Comment:
I got a little confused by this; how about aligning slightly closer to the [TSLint docs](https://palantir.github.io/tslint/rules/jsdoc-format/)?

```suggestion
    "ESLint does not support enforcing the first line of multiline JSDoc comments be empty.";
```

Reply:
Done!